config/development.toml

```diff
@@ -731,5 +731,11 @@ encryption_key = "A8EF32E029BC3342E54BF2E172A4D7AA43E8EF9D2C3A624A9F04E2EF79DC69
 [locker_based_open_banking_connectors]
 connector_list = ""
 
+[grpc_client]
+[grpc_client.dynamic_routing_client]
+host = "localhost"
+port = 7000
+enabled = false
```

Comment:
If you say enabled is `false` then shouldn't all the fields be optional? Maybe something like a tagged enum?